storage/innobase/xtrabackup/doc/source/faq.rst

```diff
@@ -2,10 +2,22 @@
 Frequently Asked Questions
 ================================================================================
 
-Does |Percona XtraBackup| 8.0 support making backups of databases in versions prior to 8.0?
+Does *Percona XtraBackup* 8.0 support making backups of databases in versions prior to 8.0?
 ====================================================================================================
 
-.. include:: .res/contents/text.pxb.8-0.txt
+*Percona XtraBackup* 8.0 does not support making backups of databases
+created in versions prior to 8.0 of |MySQL|, |Percona Server| or
+|Percona XtraDB Cluster|. The changes that |MySQL| 8.0 introduced
+in *data dictionaries*, *redo log* and *undo log* are incompatible
+with the previous versions. It is currently impossible for *Percona XtraBackup* 8.0 to also support versions prior to 8.0.
```

Comment:
Also, it is currently impossible for *Percona XtraBackup* 8.0 to support versions prior to 8.0.

Reply:
ignore.